javascripts/discourse/components/category-banner.js

```diff
@@ -51,6 +51,14 @@ export default class DiscourseCategoryBanners extends Component {
     );
   }
 
+  get displayCategoryDescription() {
+    return (
+      settings.show_description &&
+      this.category.description &&
```

Comment:
nit, but I think this will work fine too:

```suggestion
      this.category.description?.length > 0
```

Reply:
Good catch :+1: